conda/environment.yml

```diff
@@ -3,24 +3,112 @@
 
 # !!! If you are developing the website locally use the existing lock files
 # for your platform (`conda create --name cryocloud --file conda-linux-64.lock`)
-name: cryocloud
+
+name: cryointhecloud
```

Comment:
Oh sorry! When I updated this file in the hub image repo, it didn't have a name so I made one up, and used the longer version.  But I didn't see you'd already named it, and didn't mean to override you :)

I actually like your shorter name better - let's keep your shorter one and use that instead! I can update the name in the other one. We'll be typing that often, might as well use something more concise.

Reply:
cryocloud is perfect